### DISCUSSION TITLE
VSCode + SSH FS cannot use Copilot

### DISCUSSION THREAD

**Discussion by tomfriwel** [community/community#64170] [Copilot Conversations]
### Select Topic Area

Bug

### Body

When I connect remote server by the extension SSH FS of vscode, the Copilot not work.

It is available in Copilot's earlier versions.

I have been using the old version of copilot until today to find that it is no longer supported and I can only upgrade to the latest version.

_`Github Copilot OUTPUT:`_
_`[INFO] [fetch] [2023-08-18T07:51:17.386Z] Your editor's Copilot extension, version "copilot/1.61.7372", is out of date. Please install a newer version to access this service. You may need to update your editor to install the newest extension version.`_

**Comment by Akash1134** [comment]
Hi @tomfriwel ,

You must update to the latest version available. For details, please refer: #64011
Thanks!

**Comment by hsenag** [comment]
I'm surprised you were able to use 1.61 before, that hasn't been supported on the server for a long time.

Can you describe the problem you see with SSH FS? We haven't deliberately done anything to stop it working but equally we have never explicitly tried to support it either.

**Comment by tomfriwel** [comment]
It worked when I used it today, and I didn't change anything.

The current version is GitHub Copilot v1.109.386.

```
More Info
Published
2021-6-29, 22:26:17
Last released
2023-9-6, 09:44:24
Last updated
2023-9-6, 10:13:37
Identifier
github.copilot
```

**Reply by hsenag** [reply]
Thanks for the info! We weren't able to directly reproduce problems, but we have been making some improvements to the code that handles paths so it's likely some of those changes were the reason it's working now.
